src/drm/fairplay.js

```diff
@@ -24,8 +24,9 @@ const FairPlay: IDrmProtocol = class FairPlay {
    * @return {boolean} - Whether FairPlay can be play on the current environment.
    */
   static canPlayDrm(drmData: Array<PKDrmDataObject>): boolean {
-    _logger.debug('Can play DRM scheme of: ' + DrmScheme.FAIRPLAY);
-    return !!drmData.find(drmEntry => drmEntry.scheme === DrmScheme.FAIRPLAY) && !!window.WebKitMediaKeys;
+    const result = !!drmData.find(drmEntry => drmEntry.scheme === DrmScheme.FAIRPLAY) && !!window.WebKitMediaKeys;
```

Comment:
- you can use drmData.some which returns boolean
- is WebKitMediaKeys a cross browser api ? I see that there's also an api for window.MediaKeys

Reply:
WebKitMediaKeys is not cross-browser. this is why we use it to identify if the browser supports fairplay